packages/neos-ui/src/Containers/Modals/UnappliedChangesDialog/index.js

```diff
@@ -87,13 +88,15 @@ export default class UnappliedChangesDialog extends PureComponent {
                         <I18n id="Neos.Neos:Main:content.inspector.unappliedChangesDialog.button.danger"/>
                     </Button>
                     <Button
+                        id="neos-UnappliedChangesDialog-resume"
                         style="lighter"
                         onClick={this.handleResume}
                         className={`${style.button} ${style.resumeButton}`}
                         >
                         <I18n id="Neos.Neos:Main:content.inspector.unappliedChangesDialog.button.default"/>
                     </Button>
                     <Button
+                        id="neos-UnappliedChangesDialog-apply"
```

Comment:
It is more a nitpick but neos-unappliedChangesDialog-apply looks better or is the uppercase U important?

Reply:
Ok then it makes sense. Just looked a bit weird.